scripts/policy/protocols/conn/known-hosts.zeek

```diff
@@ -96,6 +96,10 @@ event Known::host_found(info: HostsInfo)
 		}
 	}
 
+const should_log_host =
```

Comment:
While it's single use, I find it adds clarity to what's otherwise a bit of a visual mouthful (to mix metaphors :P):

```  
       ! Cluster::is_enabled() ||
        Cluster::local_node_type() == Cluster::PROXY;
```